src/index.html

```diff
@@ -17,16 +17,36 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <div
+      class="search search--big"
+      data-qa="big"
+    >
+      <img
+        class="search__icon"
+        src="./images/Search.svg"
+        alt="search"
+      />
+      <input
+        class="search__input"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
```

Comment:
Consider adding `aria-label` or `aria-labelledby` attributes to the input fields to enhance accessibility for screen readers. This will provide a better description of the input's purpose.